src/messenger-shortcuts.js

```diff
@@ -171,7 +171,7 @@ function sendLike() {
 }
 
 function searchInConversation() {
-  var targetNode = getByAttr('em', 'data-intl-translation', 'Search in Conversation');
+  var targetNode = document.querySelector('._3szn._3szo ._5odt') // return the first one that matches the selector
   click(targetNode);
```

Comment:
@Code-Curious: Thanks for the fix. I'm not enthusiastic about hardcoding non-semantic DOM selectors (like `._3szn._3szo ._5odt`) into the extension. Is there any other way to do it?

Also, @sarangjo has a good point.